pkg/obc/provisioner.go

```diff
@@ -301,7 +301,9 @@ func NewBucketRequest(
 		}
 		if r.BucketClass.Status.Phase != nbv1.BucketClassPhaseReady {
 			msg := fmt.Sprintf("BucketClass %q is not ready", bucketClassName)
-			p.recorder.Event(r.OBC, "Warning", "BucketClassNotReady", msg)
+			if r.OBC != nil {
+				p.recorder.Event(r.OBC, "Warning", "BucketClassNotReady", msg)
```

Comment:
maybe I'm wrong, but I think that the panic was on the event for `MissingBucketClass` (line 299)

Reply:
good point; added nil pointer condition also on line 299.